apps/arweave/src/ar_config.hrl

```diff
@@ -43,7 +43,8 @@
 	webhooks = [],
 	max_connections = 1024,
 	max_gateway_connections = 128,
-	max_poa_option_depth = 8
+	max_poa_option_depth = 8,
+	kv_engine = ar_kv_rocksdb
```

Comment:
We also update `ar_config.erl` every time we add a new command flag because a flag can be specified in a JSON config file, but I would rather not introduce this flag at all for now, since we only support one kv backend.